### PR TITLE
DQM SMP PAG ISSUE

### PR DESCRIPTION
**Pull request by sasharma6**
Problem : Missing EwkMuDQM and EwkElecDQM sub folders in Physics directory.
Solution: After certain changes and including the modules, test have been performed with all tests passed. Getting all the histos in the above mentioned modules which were broken for long time

### CODE REVIEW

**Comment by dmitrijus** [comment]
please test

**Comment by fabozzi** [comment]
@fabozzi @prebello

**Comment by fabozzi** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
in addition to the comment from @dmitrijus - please also clean up the code by removing the commented (and presumably not working) code

**Comment by davidlange6** [comment]
-1
pending answers to minor comments

File: DQM/Physics/python/DQMPhysics_cff.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 from DQM.Physics.topJetCorrectionHelper_cfi import *
 
 dqmPhysics = cms.Sequence( bphysicsOniaDQM 
-                           #*ewkMuDQM # FIXME: broken after PR #14312
-                           #*ewkElecDQM
+                           *ewkMuDQM # FIXME: broken after PR #14312
```
Comment (sasharma6) [inline DQM/Physics/python/DQMPhysics_cff.py]: yes i will remove all the unwanted comments @fabozzi @davidlange6

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**